src/common/base/base.scss

```diff
@@ -34,8 +34,13 @@ a:hover{
     color: $brand-primary;
 }
 
-h3, h4{
-    font-size: 1em;
+h3{
+    font-size: $space-size + 4;
```

Comment:
@nickdotreid @mkruszynski Still we have some magic numbers here. Maybe a better idea is to either use em or define a `$h4-size` and `$h3-size` variables in [app.variables.scss](https://github.com/motech/motech-ui/blob/master/src/app/app.variables.scss)?